dask/bytes/core.py

```diff
@@ -48,7 +48,7 @@ def read_bytes(urlpath, delimiter=None, not_zero=False, blocksize="128 MiB",
         Chunk size in bytes, defaults to "128 MiB"
     compression : string or None
         String like 'gzip' or 'xz'.  Must support efficient random access.
-    sample : int or string
+    sample : int, string, or boolean
```

Comment:
Only False is a valid boolean here, right?

Reply:
This should have a test

Reply:
There is a check for `True` down below for backwards compatibility reasons, so I changed it to `boolean`. If that should be treated as private, I can change it back to `False`.

Reply:
That's fine, just making sure.

Reply:
Added a (very) simple test here.